genai_cookbook/nbs/2-fundamentals-unstructured-chain.md

```diff
@@ -1,6 +1,6 @@
-## Retrieval, augmentation, and generation (aka RAG Chain)
+## Retrieval, augmentation, and generation (aka RAG Agent)
 
-Once the data has been processed by the data pipeline, it is suitable for use in the RAG application. This section describes the process that occurs once the user submits a request to the RAG application in an online setting. The series, or *chain* of steps that are invoked at inference time is commonly referred to as the RAG chain.
+Once the data has been processed by the data pipeline, it is suitable for use in the retreival tool. This section describes the process that occurs once the user submits a request to the Agent application in an online setting. The series, or *chain* of steps that are invoked at inference time is commonly referred to as the Agent loop.
 
 ```{image} ../images/2-fundamentals-unstructured/3_img.png
```

Comment:
Unfortunately this image still says "RAG chain" - @epec254 do you know who made these graphics? Wonder if they can help us make one showing an agentic loop, or if we already have one in Databricks docs

Reply:
If we can't get an updated image quickly, @prithvikannan I would actually suggest not showing this image for now since it differs from the updated text below, but we should follow up on getting one